helm-repos/index.yaml

```diff
@@ -129,17 +113,11 @@ entries:
     maintainers:
     - email: eric.greer@comcast.com
       name: integrii
-    - email: jacob.martin@comcast.com
-      name: lolimjake
-    - email: ianhoegen@gmail.com
-      name: ihoegen
-    - email: jonathan_phan@comcast.com
-      name: jonnydawg
```

Comment:
dont remove!

Reply:
yeah i copied the wrong lines, but its not a big deal...